File: patterns/c-patterns/pattern91.c
// Pattern 91: ...

// * * * * *
// * * * *
// * * *
// * *
// *
// * *
// * * *
// * * * *
// * * * * *



#include <stdio.h>
#include <math.h>
int main()
{
    int size = 4;
    int i, j;

    for (i = size; i >= -size; i--)
    {
        for (j = 0; j <= abs(i); j++)
        {
            printf("* ");
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern91.js
// Pattern 91: ...

// * * * * *
// * * * *
// * * *
// * *
// *
// * *
// * * *
// * * * *
// * * * * *

let size = 4;

for (let i = size; i >= -size; i--) {
    let row = '';
    
    for (let j = 0; j <= Math.abs(i); j++) {
        row += '* ';
    }

    console.log(row);
}
